suite-common/message-system/schema/config.schema.v1.json

```diff
@@ -363,7 +363,7 @@
             "title": "Localization",
             "description": "A multilingual text localization.",
             "type": "object",
-            "required": ["en-GB", "en", "es", "cs", "ru", "ja"],
+            "required": ["en-GB", "en", "es", "fr"],
```

Comment:
looks like only en and fr is official (`languages.ts` file)

Reply:
Ok,.. removing "es" too...

Reply:
but removing anything from required fields is potentially a breaking change - some Suite version might count on the fact that it would definitely be there... I'd rather leave as mandatory everything that was already mandatory and just add FR. Or at least do a quick check that Suite never counted on those languages being there

Reply:
I've put back in all the languages that were previously required..

Reply:
ok done